packages/ember-uploader/lib/s3.js

```diff
@@ -49,11 +50,17 @@ export default Uploader.extend({
     data.type = file.type;
     data.size = file.size;
 
+    var signRequestType = this.get('signRequestType');
+    if (signRequestType !== 'GET') {
+      data = JSON.stringify(data);
+    }
+
     var settings = {
       url: get(this, 'url'),
       headers: get(this, 'headers'),
-      type: 'GET',
-      contentType: 'json',
+      type: signRequestType,
```

Comment:
Quick testing says it doesn't work that way, however, after thinking more, I'm not so sure its a great thing to cover all method types. If jQuery doesn't cover it, we probably shouldn't either.